test/server/routes/applications.test.js

```diff
@@ -133,6 +140,7 @@ describe('routes/applications.js', () => {
     return request(app)
       .post(slug)
       .set('Cookie', signedInCookie)
+      .send({ _csrf: CSRF_TOKEN })
       .send(routeData.addApplication)
```

Comment:
i think we can merge the two separate calls to `.send` with something like:

`.send({ ...routeData.addApplication, _csrf: CSRF_TOKEN })`

if it works, then you can apply it everywhere we have the double call to `.send`.

Reply:
Removed csrf send where followed by a routeData one as csrf added to latter